### PR TITLE
Remove Openstack provider code

### PR DESCRIPTION
**Pull request by aufi**
Openstack code was extracted to its own repo. Removing it from gems-pending.

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Fryguy) [inline .project]: Might I recommend putting `.project` in your global gitignore on your personal machine?  See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore  It helps avoid these kinds of things getting committed.

Comment (aufi) [inline .project]: Ah, sorry, thanks for info.

**Comment by aufi** [comment]
Code import PR https://github.com/ManageIQ/manageiq-providers-openstack/pull/14 was werged, removing WIP.

@miq-bot rm_label wip

File: lib/gems/pending/openstack/openstack_event_monitor.rb [hunk 2]
```diff
@@ -1,129 +0,0 @@
-# The OpentstackEventMonitor uses a plugin pattern to instantiate the correct
-# subclass as a plugin based on the #available? class method implemented in each
-# subclass
-require 'more_core_extensions/core_ext/hash'
```
Comment (bdunne) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: `more_core_extensions` should be listed as a runtime dependency on `manageiq-providers-openstack`

File: lib/gems/pending/openstack/openstack_event_monitor.rb [hunk 3]
```diff
@@ -1,129 +0,0 @@
...
-require 'util/extensions/miq-module'
-require 'active_support/core_ext/class/subclasses'
```
Comment (bdunne) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: `activesupport` should be listed as a runtime dependency on `manageiq-providers-openstack`

File: lib/gems/pending/openstack/openstack_event_monitor.rb [hunk 4]
```diff
@@ -1,129 +0,0 @@
-# The OpentstackEventMonitor uses a plugin pattern to instantiate the correct
-# subclass as a plugin based on the #available? class method implemented in each
-# subclass
-require 'more_core_extensions/core_ext/hash'
-require 'util/extensions/miq-module'
```
Comment (bdunne) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: `manageiq-gems-pending` should be listed in the gemfile of `manageiq-providers-openstack` with a comment that we're using `miq-module`

File: lib/gems/pending/openstack/openstack_event_monitor.rb (hunk 2 above)
Comment (aufi) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: The dependency was added, thanks for catching it.

File: lib/gems/pending/openstack/openstack_event_monitor.rb (hunk 4 above)
Comment (aufi) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: The dependency was added, thanks for catching it.

File: lib/gems/pending/openstack/openstack_event_monitor.rb (hunk 3 above)
Comment (aufi) [inline lib/gems/pending/openstack/openstack_event_monitor.rb]: The dependency was added, thanks for catching it.

**Pull request merged**